portfolio/src/main/webapp/style.css

```diff
@@ -3,17 +3,61 @@ html, body {
   font-family: helvetica, sans-serif;
 }
 
-#index{
+body{
     position: absolute;
     margin: 0px;
-    overflow: hidden;
     top: 0px;
     bottom: 0px;
     right: 0px;
     left: 0px;
-    border-style: solid;
-    border-width: 15px;
-    border-color: black;
+}
+
+/* Frame */
+.frame {
+   position: fixed;
+   z-index: 990;
+   background-color: black;
+}
+
+#frametop {
+   top: 0;
+   left: 0;
+   width: 100%;
+   height: 15px;
+}
+
+#frameright {
+   top: 0;
+   bottom: 0;
+   right: 0;
+   width: 15px;
+}
+
+#framebottom {
+   bottom: 0;
+   left: 0;
+   width: 100%;
+   height: 15px;
+}
+
+#frameleft {
+   top: 0;
+   bottom: 0;
+   left: 0;
+   width: 15px;
+}
+
+/*Each Page */
```

Comment:
nit: extra space after \/\*

Have space after \/\* and before \*\/ throughout the files